src/pages/RecipeDetail.js

```diff
@@ -1,14 +1,30 @@
+import {useState} from 'react';
+import {AiTwotoneDelete} from 'react-icons/ai';
 import {BiCategoryAlt} from 'react-icons/bi';
 import {BsClockHistory} from 'react-icons/bs';
 import {FiAlignJustify} from 'react-icons/fi';
-import {useParams} from 'react-router-dom';
+import {useParams, useNavigate} from 'react-router-dom';
 import styled from 'styled-components';
 
-export default function RecipeDetail({recipes}) {
+import Dialogue from '../components/Dialogue/Dialogue';
+
+export default function RecipeDetail({recipes, onDeleteRecipe}) {
   const params = useParams();
+  const navigate = useNavigate();
   const recipe = recipes.find(recipe => recipe.id === params.id);
```

Comment:
Then you could change this to:

```suggestion
  const recipe = recipes.find(recipe => recipe.id === id);
```